opentelemetry-api/src/opentelemetry/context/async_context.py

```diff
@@ -12,32 +12,34 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import typing  # pylint: disable=unused-import
-from contextvars import ContextVar
+try:
+    from contextvars import ContextVar
+except ImportError:
+    pass
+else:
```

Comment:
why not put this in the exception?

Reply:
Sorry, but I don't get this question.

Reply:
Hello @toumorokoshi,
Could you please explain your question?
It's not clear to me currently.
Thank you!

Reply:
Ah nevermind, I understand that the except was needed to ensure the exception doesn't raise. Got it.